src/assets/_scss/vendor/font-awesome/_bordered-pulled.scss

```diff
@@ -0,0 +1,25 @@
+// Bordered & Pulled
+// -------------------------
+
+.#{$fa-css-prefix}-border {
+  padding: .2em .25em .15em;
```

Comment:
`.2` should be written with a leading zero as `0.2`<br>`.25` should be written with a leading zero as `0.25`<br>`.15` should be written with a leading zero as `0.15`<br>Properties should be ordered border, border-radius, padding